lib/active_merchant/billing/gateways/worldpay_us.rb

```diff
@@ -74,8 +74,8 @@ def verify(credit_card, options={})
 
       private
 
-      def url
-        @options[:use_backup_url] ? self.backup_url : self.live_url
+      def url(options)
+        options[:use_backup_url] ? self.backup_url : self.live_url
```

Comment:
In the past we've coerced the type here to be something like-

```
options[:use_backup_url].to_s == "true" ? self.backup_url : self.live_url
```

so we're a little more explicit about what we're expecting vs. `"false"` also causing this to trigger. Just a thought!

Reply:
I'll change that as well to be consistent. Thanks!